### ISSUE TITLE
Трудоустройство 

### ISSUE THREAD

**Issue by Ant3333** [paralect/startup-summer#5]
Здравствуйте,будет ли осуществляться трудоустройство тех,кто успешно закончит работу над проектом и можно ли распределиться в вашу компанию в случае успешного прохождения всех этапов стажировки/работы?

**Comment by anorsich** [comment]
@Ant3333 Да, трудоустройство и распределение возможно.  Решение будет приниматься по окончанию курсов. Главное показать себя этим летом :)
